civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -15,4 +15,45 @@
         padding: 0;
         background-color: transparent;
     }
+
+    .pane-content {
+      overflow: visible !important;
+    }
+
+    #bootstrap-theme .dropdown-menu[datepicker-popup-wrap] {
+      width: 316px;
```

Comment:
Because without this value the width will be the parent's size. This value `316px` it's the value at _hrjobcontract_